### DISCUSSION TITLE
Pull multiple files from Azure Blobstorage

### DISCUSSION THREAD

**Discussion by VAluvila** [grafana/grafana-infinity-datasource#872] [Ideas]
I could not find a way to pull multiple files from Azure Blob storage using infinity. Is that not supported?
If it is supported could you please specify how? Otherwise consider this as a feature request, please

**Comment by yesoreyeram** [comment]
No it is not supported. You have to use separate queries for each file.

**Reply by kristeroksnes** [reply]
@yesoreyeram 
I was also wondering about this

I have a storage container called logs
in there is a folder called action-logs
in that folder theres json log files that i want to import to grafana.
Is that not possible with this datasource?

**Reply by VAluvila** [reply]
@kristeroksnes No. It is not possible. For my use case I switched from storage container to SQL database where multiple files will be stored and using mysql datasource I could get nice visualizations in Grafana
